MoltenVK/MoltenVK/Layers/MVKExtensions.def

```diff
@@ -95,6 +95,7 @@ MVK_EXTENSION(KHR_synchronization2,                   KHR_SYNCHRONIZATION_2,
 MVK_EXTENSION(KHR_timeline_semaphore,                 KHR_TIMELINE_SEMAPHORE,                 DEVICE,   10.11,  8.0,  1.0)
 MVK_EXTENSION(KHR_uniform_buffer_standard_layout,     KHR_UNIFORM_BUFFER_STANDARD_LAYOUT,     DEVICE,   10.11,  8.0,  1.0)
 MVK_EXTENSION(KHR_variable_pointers,                  KHR_VARIABLE_POINTERS,                  DEVICE,   10.11,  8.0,  1.0)
+MVK_EXTENSION(KHR_vertex_attribute_divisor,           KHR_VERTEX_ATTRIBUTE_DIVISOR,           DEVICE,   10.11,  8.0,  1.0)
```

Comment:
Also update list in `MoltenVK_Runtime_UserGuide.md` (by ext type then alpha order)